src/app/story/directives/hero.component.ts

```diff
@@ -46,6 +47,12 @@ import { StoryModel } from '../../shared';
              [working]="story.isPublishing" (click)="togglePublish()" orange=1>
             {{story.publishedAt ? 'Unpublish' : 'Publish'}}
           </publish-button>
+          <p *ngIf="!story.changed() && !story.publishedAt">
+            Will be queued up for publishing {{story.releasedAt ? ' at ' + formatDate(story.releasedAt) : 'immediately'}}
```

Comment:
I know it was in the requirements, but i'm not a huge fan of adding more stuff to the toolbar.  Sort of clutters the place up, and messes with whitespace (http://cav.is/s/b1gbi.jpg).

Not going to hold things up based on my opinion alone... but as an alternative to displaying this hint here:

- If publishedAt is null and releasedAt is null-or-in-the-past, button text says `Publish Now`
- If publishedAt is null and releasedAt is in-the-future, button says `Publish In X Hours` (or days, or minutes, etc).  Just a short preview of the distance between `now` and `releasedAt`
- If publishedAt is in-the-future, button has the "Private" color from the dashboard and says `Publishing in X Hours`
  - On hover, button says `Unpublish` and lets you set `publishedAt` back to null again
- If publishedAt is in-the-past, button has the "Public" color from the dashboard and says `Published`
  - On hover, button says `Unpublish` and lets you set `publishedAt` back to null again